.github/workflows/build.yml

```diff
@@ -14,6 +14,7 @@ on:
     paths-ignore:
       - '**.md'
       - '**.txt'
+  merge_group:    
```

Comment:
```suggestion
  merge_group:
    types: [checks_requested]
```

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge-group